index.bs

```diff
@@ -584,7 +589,13 @@ Registry instances are initially created with the internal slots described in th
 1. If _pair_ exists, then:
   1. Let _entry_ be _pair_.[[value]].
 1. Else:
-  1. Let _entry_ be a new registry entry record { [[Key]]: _key_, [[State]]: "fetch", [[Metadata]]: *undefined*, [[Fetch]]: *undefined*, [[Translate]]: *undefined*, [[Instantiate]]: *undefined*, [[Dependencies]]: *undefined*, [[Module]]: *undefined*, [[Error]]: *nothing* }.
+  1. Let _pipeline_ to be a new List.
+  1. Add new stage entry record { [[Stage]]: "fetch", [[Result]]: undefined } as a new element of the list _pipeline_.
```

Comment:
I'm not very happy with this construction, open to a better way to define default entries for a list. we can't not formalize it in the structure because sometimes we create it with a single entry (e.g.: installing into "ready" stage).